### DISCUSSION TITLE
Example of waiting for Provider to be Ready

### DISCUSSION THREAD

**Discussion by ytsarev** [crossplane/crossplane#2206] [Knowledge base]
There are situations when you apply provider dependant resources like ProviderConfig in an automated way right after provider installation and it fails with
```
 no matches for kind "ProviderConfig" in version "aws.crossplane.io/v1beta1"
```

To wait for the provider to be ready you can use the following command in your scripts
```
kubectl wait --for=condition=Healthy providers crossplane-provider-aws
```

**Comment by TehilaTheStudent** [comment]
but how can i do that via helm?
